From bdf87f01013da6b9c464541472b4e37777d0873b Mon Sep 17 00:00:00 2001
From: Kent Overstreet <kent.overstreet@linux.dev>
Date: Mon, 11 Dec 2023 15:12:04 -0500
Subject: [PATCH 64/74] uapi/linux/resource.h: fix include

We should't be depending on time.h; we should only be pulling in other
uapi headers.

Signed-off-by: Kent Overstreet <kent.overstreet@linux.dev>
---
 include/uapi/linux/resource.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/linux/resource.h b/include/uapi/linux/resource.h
index ac5d6a303..8c4362825 100644
--- a/include/uapi/linux/resource.h
+++ b/include/uapi/linux/resource.h
@@ -2,7 +2,7 @@
 #ifndef _UAPI_LINUX_RESOURCE_H
 #define _UAPI_LINUX_RESOURCE_H
 
-#include <linux/time.h>
+#include <uapi/linux/time_types.h>
 #include <linux/types.h>
 
 /*
-- 
2.43.0.232.ge79552d197

